virtio-devices/src/rng.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2017 The Chromium OS Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This is definitely wrong - we have changed this file after it was imported so it should be:
`// SPDX-License-Identifier: BSD-3-Clause AND Apache-2.0`

The only files that should be exclusively BSD-3-Clause are those that have never been changed after they were imported.

Reply:
done